### PR TITLE
Add seenomaly lambda to server folder 

### PR DESCRIPTION
**Pull request by beining00**
## Changes 
* add seenomaly lambda to server folder 
* seenomaly can be triggered by both API (single job)  and SQS (batch job )
## Note 
* comment out seenomaly config in the template.yml since building this is time consuming 

## TODO 
* adjust the dynamoDB update according to the job schema 
* fix the SQS calling bug

### CODE REVIEW

**Review by YinghaoMa — APPROVED** [review]

File: server/seenomaly/lambda/constants.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import os
+
+ROOT_PATH = "." #Relative path should work fine, if issues arrise set to the absoulute path of the Seenomaly containing directory.
+UNQUALIFIED_DATA_PATH = "data"
```
Comment (mt-empty) [inline server/seenomaly/lambda/constants.py:4]: What does UNQUALIFIED_DATA_PATH mean?

File: server/seenomaly/lambda/app.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import multiEventHandler 
+def handler(event, context):
+    # check event header 
+    if "httpMethod" in event and event["httpMethod"] =="POST":
```
Comment (mt-empty) [inline server/seenomaly/lambda/app.py:4]: I think we need to have a python automatic lint (just like prettier), maybe I will work on it later.

File: server/seenomaly/lambda/localTest.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import app
```
Comment (mt-empty) [inline server/seenomaly/lambda/localTest.py:1]: What exactly is this file for, Can you have some documentation?

**Review by mt-empty — CHANGES_REQUESTED** [review]
Great work.
Its best If you have someone from backend review this PR, I'm not up to date with the code base.

File: server/seenomaly/lambda/constants.py (hunk 1 above)
Comment (beining00) [reply server/seenomaly/lambda/constants.py:4]: I don't really know tbh haha, looks like something they had when developing the algorithm

Comment (reteppeter) [reply server/seenomaly/lambda/constants.py:4]: It's just the name of the folder (or subfolder) of where the data is stored. It's unqualified as it isn't valid until being combined with the root path on the next line. This is only used when testing locally.

**Comment by mt-empty** [comment]
Does the backend team plan to have `mp4` and other binary files under version control?

**Comment by reteppeter** [comment]
> Does the backend team plan to have `mp4` and other binary files under version control?
Are you taking about test data?

**Pull request merged**